### PR TITLE
Blog single page

### PR DESCRIPTION
**Pull request by wevekamola**
PR includes
 - Single blog page UI

### CODE REVIEW

File: public/wp-content/themes/glific/single.php [hunk 1]
```diff
@@ -1,12 +1,34 @@
 <?php
+
 /**
- * Template for displaying all single posts
+ * Template for displaying single posts
  *
- * @package ColoredCow
+ * @package glific
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/single.php:6]: 👏

File: public/wp-content/themes/glific/src/scss/utilities.scss [hunk 2]
```diff
@@ -190,6 +190,10 @@
 	cursor: default!important;
 }
 
+.mt-n350{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/utilities.scss]: Why do we need to explicitly create this? Please try the bootstrap class.
Also, why do we need to use 350 margin, that too in negative 🤔  ?

File: public/wp-content/themes/glific/src/scss/utilities/md.scss [hunk 3]
```diff
@@ -80,4 +80,8 @@
 			left: $value * -1;
 		}
 	}
+	
+	.mt-md-n500{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/utilities/md.scss]: Please check

**Review by Satendra-SR — COMMENTED** [review]
@wevekamola Added few comments, Please have a look.

File: public/wp-content/themes/glific/src/scss/utilities.scss (hunk 2 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/src/scss/utilities.scss]: @Satendra-SR Thanks for pointing that out. It's the requirement for the design of the blog page where we need an overlap section. Let me know if you have any more queries


Screenshot 2020-09-26 at 7 30 18 PM

File: public/wp-content/themes/glific/src/scss/utilities/md.scss (hunk 3 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/src/scss/utilities/md.scss]: 👍

**Review by Satendra-SR — APPROVED** [review]
Looks good now 👍

**Pull request merged**